MetaMask/metamask-extension
# fix: Provider padding to address button in Receive modal

## **Description**

The "Receive tokens" modal doesn't have padding around the AddressCopyButton and it looks bad.  This PR adds padding in.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24200?quickstart=1)

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. On the wallet screen, click "Receive tokens"
2. See padding around the AddressCopyButton

## **Screenshots/Recordings**



### **Before**


<img width="382" alt="SCR-20240423-kyfy" src="https://github.com/MetaMask/metamask-extension/assets/46655/4f3bdc48-43b7-4d78-a4fc-9018dc4ddd0f">


### **After**

<img width="375" alt="SCR-20240423-kybe" src="https://github.com/MetaMask/metamask-extension/assets/46655/4599a033-233f-431c-9aa3-10485ec5fab0">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
